src/components/sbb-clock/sbb-clock.stories.js

```diff
@@ -2,77 +2,35 @@ import { h } from 'jsx-dom';
 import isChromatic from 'chromatic/isChromatic';
 import readme from './readme.md';
 
-const Template = (args) => <sbb-clock {...args} />;
-
-const times = {
+const dataNow = {
   control: {
-    type: 'select',
+    type: 'date',
   },
-  options: [
-    'now',
-    '01:59:27',
-    '05:39:12',
-    '09:48:13',
-    '11:59:13',
-    '13:30:41',
-    '16:50:00',
-    '20:03:21',
-    '23:59:39',
-  ],
 };
 
-const defaultArgTypes = {
-  'initial-time': times,
-};
+const Template = (args) => <sbb-clock {...args} />;
 
 export const Default = Template.bind({});
-
-Default.argTypes = defaultArgTypes;
-Default.args = {
-  'initial-time': times.options[0],
-  paused: false,
-};
-
+Default.argTypes = { 'data-now': dataNow };
+Default.args = { 'data-now': undefined };
 Default.documentation = {
   title: 'Default',
 };
 
 export const Paused = Template.bind({});
-
-Paused.argTypes = defaultArgTypes;
-Paused.args = {
-  'initial-time': times.options[1],
-  paused: true,
-};
-
+Paused.argTypes = { 'data-now': dataNow };
+Paused.args = { 'data-now': new Date('2023-01-24T10:10:30+01:00').valueOf() };
 Paused.documentation = {
   title: 'Paused',
 };
 
-export const InitialTime = Template.bind({});
-
-InitialTime.argTypes = defaultArgTypes;
-InitialTime.args = {
-  'initial-time': times.options[1],
-  paused: false,
-};
-
-InitialTime.documentation = {
-  title: `Initial time set to ${times.options[1]}`,
-};
-
 /**
  * Stop the clock for Chromatic visual regression tests
  * and set time to given time
  */
 if (isChromatic()) {
   Default.args = {
-    'initial-time': times.options[1],
-    paused: true,
-  };
-
-  InitialTime.args = {
-    paused: true,
+    'data-now': new Date('2023-01-24T10:10:30+01:00').valueOf(),
```

Comment:
I see that the clock is displayed with 09:10:30. I guess it's coming from the +01:00 timezone.  But the question is, in which timezone should the clock be shown? As it's only for testing, this is not a big question, but I'm insecure wether the current approach is the best?

Reply:
It's only chromatic. All is ok